src/main/java/com/imsweb/validation/ValidatorContextFunctions.java

```diff
@@ -298,6 +298,42 @@ public Object fetchConfVariable(String id) throws ValidationException {
     }
 
     /**
+     * Logs the message
+     * <p/>
+     * Created on Dec 20, 2007 by depryf
+     * @param message Message to log
+     */
+    @ContextFunctionDocAnnotation(paramName1 = "message", param1 = "Message", desc = "Logs the given message.",
```

Comment:
OK, let's leave it that way if the other ones are like that. A bit confusing, but whatever :-)